templates/shared/variables/variable_list.tmpl

```diff
@@ -0,0 +1,83 @@
+<h4 class="ui top attached header">
+	{{.locale.Tr "actions.variables.management"}}
+	<div class="ui right">
+		<button class="ui primary tiny button show-modal"
+			data-modal="#edit-variable-modal"
+			data-modal-form.action="{{.Link}}/new"
+			data-modal-header="{{.locale.Tr "actions.variables.creation"}}"
+		>
+			{{.locale.Tr "actions.variables.creation"}}
+		</button>
+	</div>
+</h4>
+<div class="ui attached segment">
+	{{if .Variables}}
+	<div class="ui list">
+		{{range $i, $v := .Variables}}
+		<div class="item gt-df gt-ac gt-fw {{if gt $i 0}} gt-py-4{{end}}">
+			<div class="content gt-f1 gt-ellipsis">
+				<strong>{{$v.Title}}</strong>
+				<div class="print meta gt-ellipsis">{{$v.Content}}</div>
+			</div>
+			<div class="content">
+				<span class="color-text-light-2 gt-mr-5">
+					{{$.locale.Tr "settings.added_on" (DateTime "short" $v.CreatedUnix) | Safe}}
+				</span>
+				<button class="btn interact-bg gt-p-3 show-modal"
+					data-tooltip-content="{{$.locale.Tr "actions.variables.edit_tooltip" $v.Content}}"
+					data-modal="#edit-variable-modal"
+					data-modal-form.action="{{$.Link}}/{{$v.ID}}/edit"
+					data-modal-header="{{$.locale.Tr "actions.variables.edit"}}"
+					data-modal-dialog-variable-title="{{$v.Title}}"
+					data-modal-dialog-variable-content="{{$v.Content}}"
+				>
+					{{svg "octicon-pencil"}}
+				</button>
+				<button class="btn interact-bg gt-p-3 link-action"
+					data-tooltip-content="{{$.locale.Tr "actions.variables.delete" $v.Content}}"
```

Comment:
Why the "content" is shown in the tooltip ?

Reply:
so just display "Remove Variable" in the tooltip?